020-quarkus-http-non-application-endpoints/src/test/java/io/quarkus/qe/non_application/endpoint/NonAppEndpointNonRootPathIT.java

```diff
@@ -1,7 +1,19 @@
 package io.quarkus.qe.non_application.endpoint;
 
-import io.quarkus.test.junit.NativeImageTest;
+import io.quarkus.qe.http.non_application.endpoint.HelloResource;
+import io.quarkus.test.QuarkusProdModeTest;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-@NativeImageTest
 public class NonAppEndpointNonRootPathIT extends NonAppEndpointNonRootPathTest {
+
+    @RegisterExtension
+    static final QuarkusProdModeTest nonRootPathScenario = new QuarkusProdModeTest()
+            .setBuildNative(Boolean.getBoolean(System.getProperty(QUARKUS_PROFILE)))
+            .overrideConfigKey("quarkus.http.root-path", "/api")
+            .overrideConfigKey("quarkus.http.non-application-root-path", BASE_PATH)
+            .overrideConfigKey("quarkus.http.redirect-to-non-application-root-path", "false")
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClass(HelloResource.class)).setRun(true);
```

Comment:
I think this is a leftover. The scenario is already declared in the parent class.